test/unit/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.connectors
+
+import org.mockito.ArgumentMatchers._
+import org.mockito.Mockito._
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatestplus.mockito.MockitoSugar
+import org.scalatestplus.play.PlaySpec
+import uk.gov.hmrc.customs.emailfrontend.audit.Auditable
+import uk.gov.hmrc.customs.emailfrontend.config.AppConfig
+import uk.gov.hmrc.customs.emailfrontend.connectors.CustomsDataStoreConnector
+import uk.gov.hmrc.customs.emailfrontend.model.Eori
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.Future
+
+class CustomsDataStoreConnectorSpec extends PlaySpec with ScalaFutures with MockitoSugar {
+
+  private val mockHttp = mock[HttpClient]
+  private val mockAuditable = mock[Auditable]
+  private val mockAppConfig = mock[AppConfig]
+  private implicit val hc = new HeaderCarrier()
+  val testConnector = new CustomsDataStoreConnector(mockAppConfig, mockHttp, mockAuditable)
+
+  "CustomsDataStoreConnector" should {
+    "successfully send a query request to customs data store and return the OK response" in {
+      when(mockHttp.doPost(any(), any(), any())(any(), any())).thenReturn(Future.successful(HttpResponse(200)))
```

Comment:
can we not check for enough param values to `doPost` here instead of `any()`, so that we could have more confidence in the right values being passed to the call?